lua/lean/infoview.lua

```diff
@@ -200,8 +187,8 @@ function M.update()
 end
 
 function M.enable(opts)
+  opts.one_per_tab = opts.one_per_tab or true
```

Comment:
I believe this would be interpreted as a boolean rather than ternary expression (and would always evaluate to `true`).

Reply:
Thanks, sloppy me, yeah should be an `if` -- will fix if it's not already on your new branch.